upstream/upstream.go

```diff
@@ -189,7 +189,7 @@ func NewListWithCmp[T Backend](upstreams []T, cmp func(T, T) bool, options ...Li
 			initialScore:      1.0,
 
 			healthcheckInterval: 1 * time.Second,
-			healthcheckTimeout:  100 * time.Millisecond,
```

Comment:
Should we make this configurable? So other places using this wouldn't be changed.

Reply:
yep, sure

Reply:
oopsie :D